custom_components/cz_energy_spot_prices/translations/cs.json

```diff
@@ -19,76 +19,142 @@
                     "additional_costs_buy_gas": "Cena plynu při nákupu"
                 },
                 "data_description": {
-                    "additional_costs_buy_electricity": "Šablona pro výpočet konečné ceny po zahrnutí dodatečných nákladů. Použijte `value` pro získání aktualní spotové ceny. Například pro přídání 21 procent DPH použijte `'{{ value * 1.21 }}`.",
-                    "additional_costs_sell_electricity": "Šablona pro výpočet konečné ceny po zahrnutí dodatečných nákladů. Použijte `value` pro získání aktualní spotové ceny. Například pro přidání fixního poplatku operátora ve výši 0.25 použijte `'{{ value - 0.25 }}`.",
-                    "additional_costs_buy_gas": "Šablona pro výpočet konečné ceny po zahrnutí dodatečných nákladů. Použijte `value` pro získání aktualní spotové ceny."
+                    "additional_costs_buy_electricity": "Šablona pro výpočet konečné ceny po zahrnutí dodatečných nákladů. Použijte `value` pro získání aktualní ceny. Například pro přidání 21 procent DPH použijte `'{{ value * 1.21 }}`. Můžete použít `hour` v UTC pro spotové ceny. Pokud nezadáte žádnou šablonu, senzor nebude vytvořen.",
```

Comment:
I think the `hour` usage could use better explanation. How about: "Použíje `hour` pro výpočet ceny v dané hodině, pokud se sazba v jednotlivých hodinách liší - např. vysoký a nízký tarif."

Reply:
Fixed.